ecosystem/platform/server/app/views/it1/profile.html.erb

```diff
@@ -0,0 +1,15 @@
+<h1>It1#profile</h1>
+
+<% User.omniauth_providers.each do |provider| %>
+  <% if !current_user.authorizations.where(provider:).exists? %>
+    <%= form_with url: omniauth_authorize_path(User, provider), data: {turbo: false} do |form| %>
+      <%= form.submit "Link #{provider.to_s.titleize}", class: provider %>
+    <% end %>
+  <% end %>
+<% end %>
+
+<%= form_with do |form| %>
+  <%= form.email_field :email, autofocus: true, autocomplete: "email", value: current_user.email %>
```

Comment:
I'm not sure if the Devise email validation works when we're not using the Devise registrations controller. Needs more investigation.

Reply:
this is JS side: validation does work, yeah. In the BE I have the validation on the field as well (with stronger regex)

Reply:
I mean the confirmation email (validation was wrong terminology in the context of input boxes lol). Does it get sent when user.email is updated?

Reply:
Ah! Gotchya. Devise reconfirmable is enabled, so updating email Should update a different field until confirmed -  https://github.com/heartcombo/devise/blob/main/lib/devise/models/confirmable.rb#L27

to quickly test you can run “make mail” to run a local fake SMTP, and see if updating email will send an email or not

Reply:
Seems like it is necessary to manually call `user.send_confirmation_instructions`.